Lib/exception_group.py

```diff
@@ -0,0 +1,215 @@
+
+import sys
+
+
+class TracebackGroup:
+    def __init__(self, excs):
+        self.tb_next_map = {} # exception id to tb
+        for e in excs:
+            if isinstance(e, ExceptionGroup):
+                for e_ in e.excs:
+                    self.tb_next_map[id(e_)] = e_.__traceback__
```

Comment:
What if ‘e_’ is an EG? Shouldn’t this be recursive? (Maybe you meant ‘for e_ in e’?)

Reply:
Yes this looks wrong. I need more tests.